test/cri-containerd/layer_integrity_test.go

```diff
@@ -85,7 +85,7 @@ func Test_LCOW_Layer_Integrity(t *testing.T) {
 
 			// Validate that verity target(s) present
 			output := shimDiagExecOutput(ctx, t, podID, []string{"ls", "-l", "/dev/mapper"})
-			filtered := filterStrings(strings.Split(output, "\n"), fmt.Sprintf("dm-verity-%s", scenario.layerType))
+			filtered := filterStrings(strings.Split(output, "\n"), fmt.Sprintf("verity-%s", scenario.layerType))
```

Comment:
What is this change for?

Reply:
turned out at some point I changed the verity target naming format for scsi devices and it broke these tests. It belongs in a separate PR

Reply:
removed, pr to fix: https://github.com/microsoft/hcsshim/pull/1338